### PR TITLE
Adds ashish-amarnath as a reviewer

### PR DESCRIPTION
**Pull request by chuckha**
Signed-off-by: Chuck Ha <chuckh@vmware.com>

**What this PR does / why we need it**:

This PR adds @ashish-amarnath as a reviewer to the project. He's been a huge contributor to the project and regularly reviews code and provides feedback, features, tests and documentation. Really great to have you as an official reviewer!

```release-note
NONE
```

/assign @detiber @vincepri @randomvariable

### CODE REVIEW

**Comment by randomvariable** [comment]
You need to change `OWNERS_ALIASES`, not this file

File: OWNERS [hunk 1]
```diff
@@ -10,3 +10,4 @@ approvers:
 reviewers:
   - cluster-api-maintainers
   - cluster-api-aws-maintainers
+  - ashish-amarnath
```
Comment (detiber) [inline OWNERS]: We should probably add a new alias for reviewers

**Comment by chuckha** [comment]
ah ok, yeah I'll add a new reviewers group and add that to the owners file

**Comment by chuckha** [comment]
/cc @ashish-amarnath 

oops lol

**Comment by randomvariable** [comment]
+1

**Comment by detiber** [comment]
Adding hold for lazy consensus

/hold

**Comment by detiber** [comment]
+1 from me

/assign davidewatson
/assign d-nishi
/assign enxebre
/assign ingvagabund
/assign vincepri

**Comment by vincepri** [comment]
+1

**Comment by davidewatson** [comment]
👍

**Comment by detiber** [comment]
Removing hold, since there have been no objections to this PR
/hold cancel

**Pull request merged**